packages/at_chops/lib/src/at_platform/at_chops_keys.dart

```diff
@@ -1,27 +1,27 @@
-import 'package:at_chops/src/key/at_key_pair.dart';
-import 'package:at_chops/src/key/impl/at_encryption_key_pair.dart';
-import 'package:at_chops/src/key/impl/at_pkam_key_pair.dart';
-import 'package:at_chops/src/key/impl/at_signing_key_pair.dart';
+import 'package:at_chops/src/key/key.dart';
 
 class AtChopsKeys {
   /// Default encryption key pair
-  AtEncryptionKeyPair? atEncryptionKeyPair;
+  AsymmetricKeyPair? atEncryptionKeyPair;
 
   /// Key pair for pkam authentication. Can be legacy pkam keypair or apkam keypair for new enrollment
-  AtPkamKeyPair? _atPkamKeyPair;
+  AsymmetricKeyPair? _atPkamKeyPair;
 
   /// Key pair for data signing and verification
-  AtSigningKeyPair? atSigningKeyPair;
+  AsymmetricKeyPair? atSigningKeyPair;
 
   /// Default self encryption key
   SymmetricKey? selfEncryptionKey;
 
   /// APKAM symmetric key created during new enrollment
   SymmetricKey? apkamSymmetricKey;
 
+  /// EnrollmentId associated with pkam keys
+  String? enrollmentId;
```

Comment:
Added enrollmentId so that AtAuthKeys in at_auth can be casted to and from AtChopsKeys.